Vite 6 compatibility

Karma-vite currently shows as unsupported on Vite 6, as shown in the logs below.

Since it works correctly when forcing dependency resolution, we should bump the compatibility to allow for Vite 6.

See also: https://github.com/stryker-mutator/stryker-js/pull/5089

<details>
<summary>NPM install output</summary>

```
npm error code ERESOLVE
npm error ERESOLVE could not resolve
npm error
npm error While resolving: karma-vite@1.0.5
npm error Found: vite@6.3.3
npm error node_modules/vite
npm error   dev vite@"6.3.3" from the root project
npm error   peer vite@"^5.0.3 || ^6.0.0" from @sveltejs/kit@2.20.7
npm error   node_modules/@sveltejs/kit
npm error     dev @sveltejs/kit@"2.20.7" from the root project
npm error     peer @sveltejs/kit@"^2.0.0" from @sveltejs/adapter-auto@6.0.0
npm error     node_modules/@sveltejs/adapter-auto
npm error       dev @sveltejs/adapter-auto@"6.0.0" from the root project
npm error   4 more (@testing-library/svelte, ...)
npm error
npm error Could not resolve dependency:
npm error peer vite@"2 || 3 || 4 || 5" from karma-vite@1.0.5
npm error node_modules/karma-vite
npm error   dev karma-vite@"1.0.5" from the root project
npm error
npm error Conflicting peer dependency: vite@5.4.19
npm error node_modules/vite
npm error   peer vite@"2 || 3 || 4 || 5" from karma-vite@1.0.5
npm error   node_modules/karma-vite
npm error     dev karma-vite@"1.0.5" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
npm error
npm error
```
</details>

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "karma-vite",
-  "version": "1.0.5",
+  "version": "1.1.0",
   "description": "A karma plugin. Transform es module by using vite.",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
@@ -91,6 +91,6 @@
   },
   "peerDependencies": {
     "karma": "^6.0.0",
-    "vite": "2 || 3 || 4 || 5"
+    "vite": "2 || 3 || 4 || 5 || 6"
   }
 }

